70DaysExercises/03-Comparing Different Service Meshes.md

```diff
@@ -44,25 +45,24 @@ Cilium is a Container Networking Interface that leverages eBPF to optimize packe
 Current Version | 1.16.1 | 2.12 | N/A (it's AWS :D ) | 1.14.3 | 1.12
-Project Creators | Google, Lyft, IBM, Solo | Buoyant | AWS | Hashicorp | Cilium 
+Project Creators | Google, Lyft, IBM, Solo | Buoyant | AWS | Hashicorp | Isovalent 
 Service Proxy | Envoy, Rust-Proxy (experimental) | Linkerd2-proxy | Envoy | Interchangeable, Envoy default | Per-node Envoy
 Ingress Capabilities | Yes via the Istio Ingress-Gateway | No; BYO | Yes via AWS | Envoy | Cilium-Based Ingress
 Traffic Management (Load Balancing, Traffic Split) | Yes | Yes | Yes | Yes | Yes, but manual Envoy config required for traffic splits
 Resiliency Capabilities (Circuit Breaking, Retries/Timeouts, Faults, Delays) | Yes | Yes, no Circuit Breaking or Delays | Yes, No Fault or Delays | Yes, No Fault or Delays | Circuit Breaking, Retries and Timeouts require manual Envoy configuration, no other resiliency capabilities
-Monitoring | Access Logs, Kiali, Jaegar/Zikin, Grafana, Prometheus, LETS, OTEL | LETS, Prometheus, Grafana, OTEL | AWS X-RAY, and Cloud Watch provides these | Datadog, Jaegar, Zipkin, OpenTracing, OTEL, Honeycomb | Hubble
-Security Capabilities (mTLS, External CA) | Yes | Yes | Yes | Yes | Yes, requires manual cert creation
+Monitoring | Access Logs, Kiali, Jaegar/Zikin, Grafana, Prometheus, LETS, OTEL | LETS, Prometheus, Grafana, OTEL | AWS X-RAY, and Cloud Watch provides these | Datadog, Jaegar, Zipkin, OpenTracing, OTEL, Honeycomb | Hubble, OTEL, Prometheus, Grafana
+Security Capabilities (mTLS, External CA) | Yes | Yes | Yes | Yes | Yes, with Wireguard
 Getting Started | Yes | Yes | Yes | Yes | Yes
-Production Ready | Yes | Yes | Yes | Yes | No, Cilium CNI is Production Ready, Cilium Service Mesh isn't.
+Production Ready | Yes | Yes | Yes | Yes | Yes
 Key Features | Sidecar and Sidecar-less, Wasm Extensibility, VM support, Multi-cloud Support, Data Plane extensions | Simplistic and non-invasive | Highly focused and tight integration into AWS Ecosystem | Tight integration into Nomad and Hashicorp Ecosystem | Usage of eBPF for enhanced packet processing, Cilium Control Plane used to manage Service Mesh, No sidecars
 Limitations | Complex, learning curve | Strictly K8s, additional config for BYO Ingress | Limited to just AWS services | Storage tied to Consul and not K8s | Not a complete Service Mesh, requires manual configuration
 Protocol Support (TCP, HTTP 1.1 & 2, gRPC) | Yes | Yes | Yes | Yes | Yes
 Sidecar Modes | Sidecar and Sidecar-less | Sidecar | Sidecar | Sidecar | No sidecar
 CNI Redirection | Istio CNI Plugin | linkerd-cni | ProxyConfiguration Required | Consul CNI | eBPF Kernel processing
-Platform Support | K8s and VMs | K8s | EC2, EKS, ECS, Fargate, K8s on EC2 | K8s, Nomad, ECS, Lambda, VMs | K8s
-Multi-cluster Mesh | Yes | Yes | Yes, only AWS | Yes | Not GA
+Platform Support | K8s and VMs | K8s | EC2, EKS, ECS, Fargate, K8s on EC2 | K8s, Nomad, ECS, Lambda, VMs | K8s, VMs, Nomad
+Multi-cluster Mesh | Yes | Yes | Yes, only AWS | Yes | Yes
```

Comment:
Hey @xmulligan, does running service mesh with multi-cluster, require the usage of the Cilium Cluster-Mesh feature? Is Cluster-mesh supposed to be the same as multi-cluster service mesh? 

Just need some clarity as my understanding of Cluster-mesh was to treat all clusters running Cilium as part of a single logical flat network.

Reply:
Yes, Cluster Mesh is the way to mesh multiple clusters together. It is not just a flat network though, it can do things like [service affinity](https://isovalent.com/blog/post/cilium-release-112/#service-affinity) and [external workloads](https://isovalent.com/blog/post/cilium-release-112/#remote-kubernetes-api). Is there a certain feature you are looking for in a "multi-cluster service mesh"?

Reply:
Nah, just wanted to double check that it is a capability within Cilium Service Mesh.

All looks good to merge!